.github/workflows/terraform-plan.yml

```
@@ -7,11 +7,16 @@ on:
 env:
   AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-  GITHUB_TOKEN: ${{ secrets.REPO_TOKEN }}  
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)